src/pages/Onboard/NameTeam.vue

```diff
@@ -130,18 +138,102 @@ export default {
       await this.createLicense()
       if (!this.updateServerError) this.goToResources()
     },
-    goToResources() {
+    async accept(pt) {
+      this.loading++
+      this.accepting = true
+      try {
+        this.activeInvite = pt.id
+        const invitationId = pt.id
+        const accepted = await this.acceptMembershipInvitation(invitationId)
+        this.redirectTenant = pt.tenant.slug
+        if (accepted.accept_membership_invitation.id) {
+          this.redirectTenant = pt.tenant.slug
+          this.$apollo.queries.pendingInvitations.refetch()
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.error = true
+      }
+      this.activeInvite = null
+      this.accepting = false
+      this.loading--
+    },
+    openModal(pt) {
+      this.activeInvite = pt.id
+      this.currentInvitation = pt
+      this.dialog = true
+    },
+    async decline(id) {
+      this.loading++
+      this.declining = true
+      try {
+        this.activeInvite = id
+        const declined = await this.declineMembershipInvitation(id)
+        if (declined.delete_membership_invitation.success) {
+          this.dialog = false
+          this.$apollo.queries.pendingInvitations.refetch()
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.error = true
+      }
+      this.declining = false
+      this.activeInvite = null
+      this.loading--
+    },
+    async goToResources() {
       this.revealNameInput = false
       this.revealUrlInput = false
       this.revealConfirm = false
 
+      await this.setCurrentTenant(
+        this.redirectTenant ?? this.tenantChanges.slug ?? this.tenant.slug
+      )
+
       this.$router.push({
         name: 'onboard-resources',
         params: { tenant: this.tenant.slug }
       })
     }
   },
-  apollo: {}
+  apollo: {
+    pendingInvitations: {
+      query: require('@/graphql/Tenant/pending-invitations-by-email.gql'),
+      variables() {
+        return {
+          email: this.user.email
+        }
+      },
+      async result({ data, loading }) {
+        if (loading || !data || !data.pendingInvitations) return
+        // We filter this because we don't want to show invitations
+        // to tenants we're already in...
+        // This is due to a bug(feature?) in the back end that allows
+        // users to be invited to tenants they're already part of
```

Comment:
@cicdw bug or feature? :)

Reply:
Haha good question - I think at this exact moment it's a feature... main reason being that the UI doesn't currently display error messages if the backend fails to create an invite (it will just say "Something went wrong"), and so if we prevented it, it would look like something is broken to the user when in fact there's nothing intrinsically wrong with creating an invite to a tenant